rules/RESPONSE-951-DATA-LEAKAGES-SQL.conf

```diff
@@ -194,7 +194,7 @@ SecRule TX:sql_error_match "@eq 1" \
     ver:'OWASP_CRS/3.0.0',\
     severity:'CRITICAL',\
     chain"
-    SecRule RESPONSE_BODY "@rx (?i)(?:Exception (condition )?\d+\. Transaction rollback\.)" \
+    SecRule RESPONSE_BODY "@rx (?i)Exception (condition )?\d+\. Transaction rollback\." \
```

Comment:
I copied this as is but the parenthesis most likely should be escaped.
That said, I tried to find this pattern online and failed. Anyone has any links?